packages/account/src/Components/icon-with-message/icon-with-message.jsx

```diff
@@ -1,22 +1,49 @@
 import PropTypes from 'prop-types';
 import * as React from 'react';
-import { Icon, Text } from '@deriv/components';
+import classNames from 'classnames';
+import { Icon, Text, Button } from '@deriv/components';
+import { isMobile, PlatformContext } from '@deriv/shared';
+import { localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+
+const IconWithMessage = ({ icon, message, has_button, toggleAccountsDialog }) => {
+    const { is_dashboard } = React.useContext(PlatformContext);
 
-const IconWithMessage = ({ icon, message, more_content }) => {
     return (
-        <div className='da-icon-with-message'>
+        <div className={classNames('da-icon-with-message', { 'da-icon-with-message-full-width': is_dashboard })}>
             <Icon icon={icon} size={128} />
-            <Text className='da-icon-with-message__text' as='p' color='general' size='s' line_height='m' weight='bold'>
+            <Text
+                className='da-icon-with-message__text'
+                as='p'
+                color='general'
+                size={isMobile ? 'xs' : 's'}
+                line_height='m'
+                weight='bold'
+            >
                 {message}
             </Text>
-            {more_content}
+            {has_button && (
+                <Button
+                    primary
+                    onClick={() => {
+                        toggleAccountsDialog();
+                    }}
```

Comment:
Just an improvement suggestion
```suggestion
                    onClick={toggleAccountsDialog}
```